plugins/modules/vmware_vm_storage_policy.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Ansible Project
+# Copyright: (c) 2020, Dustin Scott <sdustin@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_vm_storage_policy
+short_description: Create vSphere storage profiles.
+description:
+- A vSphere storage policy defines metadata that describes storage requirements
+  for virtual machines and storage capabilities of storage providers.
+- Currently, only tag-based storage policy creation is supported.
+version_added: '2.9'
```

Comment:
```suggestion
version_added: '2.10'
```

FYI, there is an on-going discussion related to version_added. I would recommend to keep it like this. Once we have concerte discussion we will make changes to all modules.